deploy/charts/cert-manager/Chart.template.yaml

```diff
@@ -1,6 +1,5 @@
 apiVersion: v1
 name: cert-manager
-# The version and appVersion fields are set automatically by the release tool
 version: v0.1.0
```

Comment:
Why has this line been removed? Is this no longer true?

Reply:
Is is still true, but I didn't really see any value to the line. I'll just add it back

Reply:
Yep, we got a non-insignificant number of issuers/PRs/comments around it before there was a comment (even when there was one!) https://github.com/jetstack/cert-manager/pull/2162